README.md

```diff
@@ -1,37 +1,69 @@
-# ldcli
+# LaunchDarkly CLI
 
-## Commands
+The LaunchDarkly CLI helps you manage your feature flags wherever you are, whether that's in the webapp, your terminal, or your IDE.
```

Comment:
Are we going to have webapp features in the cli?

Reply:
this is for sure just placeholder text but this is just like a marketing-type phrase, "you can manage LD in the UI/website AND the CLI"